src/jagdish/FrequencyofChar.java

```diff
@@ -0,0 +1,40 @@
+package jagdish;
+
+import java.util.Scanner;
+
+public class FrequencyofChar {
+
+	int getCountofChar(String name, char ch) {
+		int count = 0;
+		for (int index = 0; index < name.length(); index++) {
+			if (ch == name.charAt(index))
+				count++;
+		}
+		return count;
+	}
+
+	void printFirstNonRepeatingChar(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			int cnt = getCountofChar(input, input.charAt(index));
+			if (cnt == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) != lastIndexOf(ch) logic to avoid one more for loop